capsules/detector_text_openvino/licenses/code.LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright 2020 Dilili Labs, Inc.
```

Comment:
-> Aotu

Reply:
Keep as is. We'll change the licenses across the zoo in one go in a separate PR.